refactor: change drop zone background while dragging

dated opened the pull request:
<!--
Thanks for your interest in the project. Bugs filed and PRs submitted are appreciated!

Please make sure you're familiar with and follow the instructions in the [contributing guidelines](https://ark.dev/docs/program-incentives/guidelines/contributing).

Please fill out the information below to expedite the review and (hopefully) merge of your pull request!
-->

## Summary

<!-- What changes are being made? -->

<!-- Why are these changes necessary? -->

<!-- How were these changes implemented? -->

## Checklist

https://app.clickup.com/t/h73ajj

<!-- Have you done all of these things?  -->

-   [x] My changes look good in both light AND dark mode
-   [x] The change is not hardcoded to a single network, but has multi-asset in mind
-   [x] I checked my changes for obvious issues, debug statements and commented code
-   [ ] Documentation _(if necessary)_
-   [x] Tests _(if necessary)_
-   [x] Ready to be merged

<!-- Feel free to add additional comments. -->

dated committed:
refactor: change drop zone background while dragging

faustbrian commented:
Task linked: [CU-h73ajj [Misc] missing background color of import profile drop zone](https://app.clickup.com/t/h73ajj)

boldninja committed:
chore: optimize 2 SVG(s)

Optimized SVG(s):
- src/app/assets/svg/dark/dwe-file.svg
- src/app/assets/svg/dark/json-file.svg

dated committed:
test: coverage

dated committed:
chore: update snapshots

dated committed:
wip

dated committed:
refactor: use the same icon for dark/light mode

dated committed:
chore: update file icons
